css/style.css

```diff
@@ -25,9 +25,14 @@ header {
     padding-left: .9375rem;
 }
 
-header h1 {
-    font-family: 'Knewave', cursive;
```

Comment:
Btw, I guess we can now delete the [Knewave import](https://github.com/katspaugh/wavesurfer.js/blob/gh-pages/css/style.css#L1), too.

Reply:
done.